work/conf/schema.xml

```diff
@@ -24,6 +24,8 @@
   <field name="mbid" type="mbid" indexed="true" stored="true" required="true"/>
   <field name="name" type="string" indexed="false" stored="true"/>
   <field name="ngram" type="ngram" indexed="true" stored="false" multiValued="true"/>
+  <field name="recording" type="text_mult" indexed="true" stored="false" multiValued="true"/>
```

Comment:
I'm not too versed on the schema bits, but does this expression allow multiple values for recording and recording id? And are they stable? Meaning that you can always find the right recording id for a given recording text?

Reply:
Since its a list of recordings, what sir gives to solr is all the recording ids concatanated. That is why this is multivalued.

So in simple terms if the relation list has 2 recordings with ids 1 and 2, it will pass [1, 2].

Reply:
What we are trying to do with the above is that if we have a work with a recording with a particular id, if we search for that recording id, it will return the appropriate work result.

Reply:
Similar example written by @mineo is https://github.com/samj1912/mbsssss/blob/ca6d9b9755d016b3a31577b8c5936e4a635b406d/work/conf/schema.xml#L14